website/content/docs/nia/compatibility.mdx

```diff
@@ -7,28 +7,26 @@ description: >-
 
 # Compatibility
 
-## Consul
-
-Below are the supported Consul versions with compatible Consul-Terraform-Sync versions. The latest Consul-Terraform-Sync binary targets supporting the latest patch version of the three most recent Consul minor versions.
-
-| Consul Version               | Compatible Consul-Terraform-Sync Version |
-| ---------------------------- | ---------------------------------------- |
-| 1.8+                         | 0.1+                                     |
-
-## Terraform
-
-Consul-Terraform-Sync is compatible with the following Terraform OSS versions:
-
-| Consul-Terraform-Sync | Compatible Terraform Version |
-| --------------------- | ---------------------------- |
-| 0.2+                  | 0.13 - 1.0                   |
-| 0.1                   | 0.13 - 0.14                  |
-
-## Terraform Cloud
-
-Consul-Terraform-Sync integration with Terraform Cloud is supported for the following:
-
-| Consul-Terraform-Sync Enterprise | Terraform Cloud            | Version            |
-| -------------------------------- | -------------------------- | ------------------ |
-| 0.4+                             | Terraform Cloud            | Latest             |
-| 0.3+                             | Terraform Enterprise       | v202010-2 - Latest |
+## Consul-Terraform-Sync
+
+Below are the supported Consul, Terraform, and Vault versions with compatible Consul-Terraform-Sync versions.
+
+|                                                                                                    | **CTS OSS**<br/><span style={{color:'#626873'}}><sub>v0.1+</sub></span> | **CTS Enterprise**<br/><span style={{color:'#626873'}}><sub>v0.1+</sub></span> |
```

Comment:
Here is the only guidance I can find on use of inline styles:
https://github.com/hashicorp/engineering-docs/blob/main/writing/markdown.md#inline-styles
It doesn't address the usage in the table, but I think we'd be better off using the default table styles (left alignment, no font-color, etc.) so that we create a consistent look and feel. Also, custom styles may conflict with future design refreshes.
I would replace the checkmarks and exes with the supported version numbers from the column headers.
Also, I think we should standardize on representing versions without the 'v' in front of the number. It looks cleaner.

Reply:
Thanks for this context! I posted in the [#proj-consul-docs channel ](https://hashicorp.slack.com/archives/C011HP57U0Y/p1641487063001000) hoping for a response from Bryce on the styling to see if the team is looking to make an update, if not I will revert back to left alignment! I will take a look at the other feedback and make some revisions, thanks for taking the time to review!